bus/fabric_endpoint.go

```diff
@@ -4,301 +4,301 @@
 package bus
 
 import (
-    "encoding/json"
-    "fmt"
-    "github.com/go-stomp/stomp/frame"
-    "github.com/vmware/transport-go/log"
-    "github.com/vmware/transport-go/model"
-    "github.com/vmware/transport-go/stompserver"
-    "strings"
-    "sync"
+	"encoding/json"
+	"fmt"
+	"github.com/go-stomp/stomp/v3/frame"
+	"github.com/vmware/transport-go/log"
+	"github.com/vmware/transport-go/model"
+	"github.com/vmware/transport-go/stompserver"
+	"strings"
+	"sync"
 )
 
 const (
-    STOMP_SESSION_NOTIFY_CHANNEL = TRANSPORT_INTERNAL_CHANNEL_PREFIX + "stomp-session-notify"
+	STOMP_SESSION_NOTIFY_CHANNEL = TRANSPORT_INTERNAL_CHANNEL_PREFIX + "stomp-session-notify"
 )
 
 type EndpointConfig struct {
-    // Prefix for public topics e.g. "/topic"
-    TopicPrefix           string
-    // Prefix for user queues e.g. "/user/queue"
-    UserQueuePrefix       string
-    // Prefix used for public application requests e.g. "/pub"
-    AppRequestPrefix      string
-    // Prefix used for "private" application requests e.g. "/pub/queue"
-    // Requests sent to destinations prefixed with the AppRequestQueuePrefix
-    // should generate responses sent to single client queue.
-    // E.g. if a client sends a request to the "/pub/queue/sample-channel" destination
-    // the application should sent the response only to this client on the
-    // "/user/queue/sample-channel" destination.
-    // This behavior will mimic the Spring SimpleMessageBroker implementation.
-    AppRequestQueuePrefix string
-    Heartbeat             int64
+	// Prefix for public topics e.g. "/topic"
+	TopicPrefix string
```

Comment:
can you make sure your `go fmt` works? seems like these un-indented lines are against `go fmt` behaviors

Reply:
so it seems that some of this earlier code was never formatted using `go fmt`, I turned it on globally on the project and voila. a ton of updated formatting.